### ISSUE TITLE
APK /  Android Support

### ISSUE THREAD

**Issue by Bixel925** [kenkawakenkenke/tempescope#9]
I haven't found anything on how to get Tempescope to work with Android. Is there a simple way to use Tempescope with an Android device?

**Comment by ATamalonis** [comment]
I think you'd have to write your own app on Android; last I checked there was no Android app, but maybe someone has made one. It uses Bluetooth, so there should be some generic pre-made apps to talk to Bluetooth devices and activate their outputs which could serve as a starting point or proof of concept before you start writing a lot of code. I've looked at one by Adafruit for iOS before, but didn't use it much.

**Comment by Bixel925** [comment]
Well nuts. I'm afraid that I've no idea where to even begin with writing an Android app.

**Comment by LukeSkywalker856** [comment]
Yeah well me too, I know fucking shit about writing programs for apps. For me to write an app would be like, getting my own fuel for my car, but first, I'd have to make a oil drill rig, before that, I'd have to get steal to make the rig, I'd have to make the tools to make the rig, and so on and so forth. So yeah, this is all pointless for me.
